addOns/formhandler/src/main/javahelp/org/zaproxy/zap/extension/formhandler/resources/help/contents/FormHandlerHelp.html

```diff
@@ -68,6 +68,8 @@ <H3>Modifying An Existing Field</H3>
  
 		<H3>Removing Fields</H3>
 			<p>
+				Note: users are better off disabling fields they don't want to use. Missing defaults will be restored
+				at the next start of ZAP or update of the add-on.<br>
```

Comment:
I couldn't think of a different way to handle this. If a user deletes all the defaults from a particular version of the add-on (or PR/commit) we'd have no way to know what state things are in.

Reply:
Unless we have a "deleted" section in the config I guess.... but that seems like a lot of work for questionable gain.

I suppose another option is to add some text to the removal confirmation dialog.

Reply:
If default fields are deleted, maybe we can disable them in the config? That way, even if they are added back the next time ZAP is started, they will be disabled by default.

Reply:
We talked a week or so ago in IRC about allowing to reset options panels individually, IMO that's better than always adding default configs (if the user wants to delete everything it can restore the defaults itself).

Reply:
So say like I dunno three releases from now each has new "defaults" how do I know what should and shouldn't be added? Just never adding them doesn't seem like the right move.

Reply:
This isn't limited to here either I guess. It would also be applicable for replacer, global excludes, network passthru, I dunno.....

Reply:
New defaults can be added through the versioning of the options.

Reply:
I guess we could version the config for every set of defaults 🤷‍♂️

Reply:
:grinning:

Okay I'll revise it slightly.